test/e2e/tests/confirmations/transactions/gas-fee-tokens-eip-7702.spec.ts

```diff
@@ -54,6 +54,7 @@ describe('Gas Fee Tokens - EIP-7702', function (this: Suite) {
 
         const transactionConfirmation = new TransactionConfirmation(driver);
         await transactionConfirmation.clickAdvancedDetailsButton();
+        await transactionConfirmation.closeGasFeeToastMessage();
```

Comment:
:information_source:  in this case, this step is done in the spec level, because this only happens for this specific test (edit gasless tx + eip7702), so it would make no sense to add the guard affecting all specs in confirmations.
As a general rule though, we should mitigate flakiness at the page class level